content/en/docs/tasks/run-application/access-api-from-pod.md

```diff
@@ -42,10 +42,11 @@ securely with the API server.
 
 ### Directly accessing the REST API
 
-While running in a Pod, the Kubernetes apiserver is accessible via a Service named
-`kubernetes` in the `default` namespace. Therefore, Pods can use the
-`kubernetes.default.svc` hostname to query the API server. Official client libraries
-do this automatically.
+While running in a Pod, the Kubernetes API server is referenced by the environment
```

Comment:
Hi @olix0r . I like some of the suggested edits. What do you think of this update:

While running in a Pod, your container can create an HTTPS URL for the Kubernetes API server by fetching the  `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT_HTTPS` environment variables.